.github/ISSUE_TEMPLATE/improve-existing-docs.yml

```
@@ -0,0 +1,64 @@
+name: Improve existing docs
+about: Make a suggestion to improve our existing documentation.
+title: ''
+labels: "content"
+inputs:
+- type: description
+  attributes:
+    value: 'HUBBERS BEWARE! THE GITHUB/DOCS REPO IS PUBLIC TO THE ENTIRE INTERNET. OPEN AN ISSUE IN GITHUB/DOCS-CONTENT INSTEAD.
+
+           For questions, ask in Discussions: https://github.com/github/docs/discussions'
```

A better way to write this in YAML is to use the [multi-line string support](https://yaml-multiline.info).

```suggestion
    value: |
      HUBBERS BEWARE! THE GITHUB/DOCS REPO IS PUBLIC TO THE ENTIRE INTERNET. OPEN AN ISSUE IN GITHUB/DOCS-CONTENT INSTEAD.

      For questions, ask in Discussions: https://github.com/github/docs/discussions
```